docs/buckets.rst

```diff
@@ -0,0 +1,62 @@
+Buckets
+#######
+
+Buckets are a way to group users around some data.
+
+Basically has got an and a list of principals that can administrate
+it.
+
+A bucket can be created using a PUT on the bucket URI:
+
+.. code-block:: http
+
+   > PUT /buckets/servicedenuages HTTP/1.1
+   < 201 Created
+
+    {
+      "id": "servicedenuages",
+      "principals": ["email:natim@example.com"]
```

Comment:
I don't think we want fined permissions on the bucket itself, do you have a usecase in mind that needs it?